addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -108,13 +108,12 @@ function changeOption(optionName, weName = '', optionTooltipLabel = '', position
     };
 }
 
-function selectNested(trigger, optionName, alt_trigger = null, optionTooltipLabel = '', position = "top", allowPalette = false) {
+function selectNested(trigger, optionName, altTrigger = null, optionTooltipLabel = '', position = "top", allowPalette = false) {
```

Comment:
did you consider changing the API of this function as well? i.e. remove altTrigger, and force it to be put inside the trigger with a comma

Reply:
I think it's not really useful. The function is only used once ... for me it could even disappear. the trigger depends on the alt_trigger which depends on the option_block which depends on the noPalette which depends on ... I can just rename it something else if it's the altTrigger that bothers. In any case, it's not really the subject of the PR to refactor this function :)

Reply:
ok